### PR TITLE
Display Farmer

### PR DESCRIPTION
**Pull request by quansenB**
# Description

A user can see all the information about one farmer

## Checklist

Remove any items which are not applicable.

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works AND the tests pass

### CODE REVIEW

**Comment by daquinons** [comment]
[Display farmer info](https://trello.com/c/xa36lI1C/79-display-farmer-info)

File: src/components/App/App.js [hunk 1]
```diff
@@ -2,31 +2,60 @@
...
   }, [user]);
 
+  useEffect(() => {
+    if (user && needsUpdate) {
+      updateFarmers();
+      setNeedsUpdate(false);
+    }
+  }, [user, needsUpdate]);
+
+  const updateFarmers = () => {
```
Comment (quansenB) [inline src/components/App/App.js]: Would be great to change our data locally upon edit and remove to circumvent bad connection issues. Its fine like this for MVP though

File: src/components/pages/DisplayFarmer/GuarantorTab.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { Grid, Header } from 'semantic-ui-react';
+
+export default function GuarantorTab(props) {
+  const data = props.farmer.guarantor;
+  console.log(data);
```
Comment (Pav0l) [inline src/components/pages/DisplayFarmer/GuarantorTab.js]: hehe, we don't want to be logging this 😄

File: src/components/pages/DisplayFarmer/FamilyTab.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Grid, Header } from 'semantic-ui-react';
+
+export default function FamilyTab(props) {
+  const data = props.farmer.familyInfo;
+  console.log(data);
```
Comment (Pav0l) [inline src/components/pages/DisplayFarmer/FamilyTab.js]: well, hello 😄

File: src/components/pages/DisplayFarmer/DisplayTabTemplate.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```
Comment (Pav0l) [inline src/components/pages/DisplayFarmer/DisplayTabTemplate.js]: This file can be deleted

**Pull request merged**